packages/react-components/react-avatar/src/components/AvatarGroup/useAvatarGroupStyles.ts

```diff
@@ -10,35 +11,78 @@ export const avatarGroupClassNames: SlotClassNames<AvatarGroupSlots> = {
 
 export const extraAvatarGroupClassNames = {
   popoverSurfaceItem: 'fui-AvatarGroup__popoverSurfaceItem',
+  popoverSurfaceContainer: 'fui-AvatarGroup__popoverSurfaceContainer',
 };
 
 /**
- * Styles for the root slot
+ * Styles for the root slot.
  */
 const useStyles = makeStyles({
-  root: {
-    // TODO Add default styles for the root element
+  base: {
+    display: 'inline-flex',
+    position: 'relative',
   },
+});
+
+/**
+ * Styles for the PopoverTrigger.
+ */
+const usePopoverTriggerStyles = makeStyles({
+  base: {
+    flexShrink: 0,
+    minWidth: 0,
+    minHeight: 0,
+    maxWidth: 'unset',
+    maxHeight: 'unset',
+    ...shorthands.padding(0),
+  },
+  pie: {
+    backgroundColor: 'transparent',
+    ...shorthands.borderColor('transparent'),
+    ':focus': {
+      backgroundColor: 'transparent',
+    },
+  },
+});
 
-  // TODO add additional classes for different states and/or slots
+const usePopoverSurfaceStyles = makeStyles({
+  base: {
+    [`& > .${extraAvatarGroupClassNames.popoverSurfaceContainer}`]: {
+      listStyleType: 'none',
```

Comment:
It looks like there might be some a11y concerns with using `list-style-type: none`:
https://developer.mozilla.org/en-US/docs/Web/CSS/list-style#accessibility_concerns

I think it boils down to adding `role="list"` on the `ul` and `role="listitem"` on the `li`s